examples/tutorials/translations/português/Parte 07 - Aprendizado Federado com Base de Dados Federado.ipynb

```diff
@@ -0,0 +1,217 @@
+{
```

Comment:
<p>Just a suggestion. I don't think <em>"Federado"</em> is in agreement with <em>"Base"</em>. A better alternative is <em>"Base de Dados Federada"</em> or <em>"Conjunto de Dados Federado"</em>.</p><p> </p><blockquote>Em português, federado não está nem em concordância com <em>Base</em> nem com <em>Dados. </em>Se estiver se referindo à base, Base Federada, se aos dados, Base de Dados federados. Poderia ser também Conjunto de Dados Federado, ou Dataset federado.</blockquote><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/2977/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Indeed!